web/css/events.css

```diff
@@ -24,6 +24,7 @@
   font-size: 0.85em;
   color: #ccc;
   padding: 0 3px;
+  left-margin: 40px; 
```

Comment:
Almost there! The proper syntax here is going to be `margin-left`

Reply:
How embarassing!. 🤦‍♀️ Serves me right for trying to rush it during a break at work.

Reply:
No worries!  I think we all do the same sort of thing pretty much daily 👍 

Thanks for your contribution!